apps/onboarding/src/app/result/page.tsx

```diff
@@ -82,34 +84,53 @@ const Volumn = styled.p`
 `;
 
 const Result = () => {
+  const pathname = usePathname();
   const searchParams = useSearchParams();
+  const drinkType = searchParams.get('drinkType');
   const glasses = Number(searchParams.get('glasses'));
   const { name, svg, description, mainColor } = getLevelDetails(glasses);
+  const { isLoading, data } = useQuery(['result'], () => {
+    return axios
+      .get(
+        `https://sulsul.app/api/v1/drinkingLimit/share?drinkType=${drinkType}&glass=${glasses}`
+      )
+      .then((response) => response.data);
+  });
+
+  const shareKakao = () => {
+    window.Kakao.Share.sendScrap({
+      requestUrl: `https://sulsul.app${pathname}?glasses=${glasses}`,
+    });
+  };
+
+  if (isLoading) return <div>loading...</div>;
 
   return (
     <Page>
       <Heading2>당신은...</Heading2>
       <ResultCard name={name} svg={svg} description={description} mainColor={mainColor} />
       <Heading3>다른 술은 얼마나 마실 수 있을까?</Heading3>
       <DrinkLists>
-        {Object.values(DrinkResDrinkTypeEnum).map((alcohol) => {
-          const { name, svg, volumn } = AlcoholDetails[alcohol];
+        {data.otherDrinks.map((drink: DrinkRes) => {
+          const { drinkType, glass } = drink;
+          const { name, svg, volumn } =
+            AlcoholDetails[drinkType as keyof typeof AlcoholDetails];
           return (
-            <ListItem key={alcohol}>
+            <ListItem key={drinkType}>
               <DrinkDetail>
                 <Drinks src={svg} />
                 <div>
                   <Heading5>{name}</Heading5>
                   <Volumn>{volumn}도</Volumn>
                 </div>
               </DrinkDetail>
-              <Heading5>8잔</Heading5>
+              <Heading5>{glass}잔</Heading5>
             </ListItem>
           );
         })}
       </DrinkLists>
       <ButtonWrapper>
-        <Button type="button" css={{ width: '100%' }}>
+        <Button type="button" css={{ width: '100%' }} onClick={shareKakao}>
```

Comment:
Button의 type은 기본값이 "button"이고 submit으로 수정할 수 있게 하면 어떨까요? 빼먹는 상황을 방지할 수 있을 거 같아요

```suggestion
        <Button css={{ width: '100%' }} onClick={shareKakao}>
```

Reply:
그것도 고민을 하긴 했는데, 내부에서 합의가 되면 상관이 없는데 가져다가 쓰는 입장에선 Button의 기본 동작을 당연히 submit이라고 인식하고 있지 않을까 싶어서...!